spark/interpreter/src/main/resources/interpreter-setting.json

```diff
@@ -227,7 +232,8 @@
     },
     "editor": {
       "language": "r",
-      "editOnDblClick": false
+      "editOnDblClick": false,
+      "completionSupport": true
```

Comment:
Should be false for SparkRInterpreter, as the completion method of SparkRInterpreter always return empty list

Reply:
Sure, totally missed it, let me make that change.